### DISCUSSION TITLE
Sort by state in Entity Filter Card

### DISCUSSION THREAD

**Discussion by mandrill-pie** [home-assistant/frontend#17907] [Dashboard]
Hello!
I wonder if there's any discussion on the topic of sorting by state in the entity-filter card?
My most simple scenario is a list of devices and battery percentages, sorted ascending by battery percentage.

<img width="482" alt="batteries" src="https://github.com/home-assistant/frontend/assets/3681453/c1b70256-4da9-4c8b-9766-c0b5b5c18fdb">

If there is such a discussion, I'd like to go there and upvote. If not, um, why not? :) I was surprised to find this missing, this looks like it could have a lot of uses, I think.
There are solutions using custom cards, it seems, but I'm personally not interested in those.

Also, please point me to any type of core HA device list card that can do sorting by state, if it exists, maybe I can work from there.
Thank you!

P.S. First post here, I hope I'm in the right category, this sounds like a "Dashboard" question to me, but maybe it isn't...

**Comment by karwosts** [comment]
Someone did actually implement this in a PR once, but reviewers said it wasn't a good fit for the core cards and suggested the author to create a custom card instead. 

So it sounds like that's probably the way you'll have to go if you want this feature.

**Comment by mandrill-pie** [comment]
Thank for the reply. I hope this can stay here for other people looking for how to sort/order entity lists by state. I don't know why I thought this would surely be a part of entity or entity filter cards. Maybe people don't sort entities by state. Or maybe they don't mind using custom cards.
My use cases would be things like battery percentages, a list of temperatures/pressures/humidities or other numerical values. It could maybe also bring order to a list where non-numerical values are also mixed in, such as Unavailable.

**Comment by ildar170975** [comment]
> My most simple scenario is a list of devices and battery percentages, sorted ascending by battery percentage.

I am using a custom auto-entities card for this. Has plenty of possibilities.
